app/controllers/teacher_sets_controller.rb

```diff
@@ -66,7 +66,9 @@ def show
     render json: {
       :teacher_set => @set,
       :active_hold => @active_hold,
-      :user => current_user
+      :user => current_user,
+      #Display's only  status 'new' holds count in teacher set page.
```

Comment:
Maybe rephrase to something like:
\# Count the number of new holds on this teacher set, and pass through so the front end can estimate availability.